syft/frameworks/torch/tensors/abstract.py

```diff
@@ -9,6 +9,38 @@ class AbstractTensor(ABC):
     This is the tensor abstraction.
     """
 
+    def __str__(self) -> str:
```

Comment:
This should call __repr__ or vice versa to reuse code

Reply:
I agree!